From 276c45d0663124c9ccab0ddc8b97fb5cdc8b3867 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 31 Jul 2018 16:53:04 +0800
Subject: [PATCH] configure.ac: remove prog test of xmlto and asciidoc

The prog of xmlto and asciidoc were used to generate documents,
and since the doc module has already been disabled, so we should
remove the prog test.

Upstream-Status: Inappropriate [workaround]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Modify to apply to v 2.17.8.

Signed-off-by: Joe Slater <joe.slater@windriver.com>
---
 configure.ac | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/configure.ac b/configure.ac
index da8cf898..17ece200 100644
--- a/configure.ac
+++ b/configure.ac
@@ -57,24 +57,6 @@ IT_PROG_INTLTOOL([0.35.0])
 
 dnl ****** END ****************************************
 
-AC_PATH_PROG([ASCIIDOC], [asciidoc], [no])
-[if test "$ASCIIDOC" = "no"]
-[then]
-    [echo "The asciidoc program was not found in the search path. Please ensure"]
-    [echo "that it is installed and its directory is included in the search path."]
-    [echo "Then run configure again before attempting to build libreport."]
-    [exit 1]
-[fi]
-
-AC_PATH_PROG([XMLTO], [xmlto], [no])
-[if test "$XMLTO" = "no"]
-[then]
-    [echo "The xmlto program was not found in the search path. Please ensure"]
-    [echo "that it is installed and its directory is included in the search path."]
-    [echo "Then run configure again before attempting to build libreport."]
-    [exit 1]
-[fi]
-
 AC_ARG_WITH(bugzilla,
 AS_HELP_STRING([--with-bugzilla],[use Bugzilla plugin (default is YES)]),
 LIBREPORT_PARSE_WITH([bugzilla]),
-- 
2.25.1

